core/src/Cache/CredentialType.cs

```diff
@@ -29,8 +29,8 @@ namespace Microsoft.Identity.Core.Cache
 {
     internal enum CredentialType
     {
-        IdToken,
-        AccessToken,
-        RefreshToken
+        idtoken,
```

Comment:
this values are part of cache format schemas

Reply:
(agree) pls. take in the code style discussion

---
In reply to: [199545185](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199545185) [](ancestors = 199545185)

Reply:
Not sure about this one. Are you saying that the token cache keys / value strings have these names in them? In that case, the code is brittle - I can come in and rename these enums and I'll break the cache because how would I know these are used as strings as well?